lib/dialyzer/test/behaviour_SUITE_data/src/callbacks_and_specs/my_callbacks_wrong.erl

```diff
@@ -28,13 +28,13 @@
 callback_init(Parent) -> #state{parent = Parent}. %% Wrong return
 
 -spec callback_cast(state(), pid() | atom(), cast_message()) ->
-			   {'noreply' | 'reply', state()}. %% More generic spec
+			   {'reply', state()}. %% Non-overlapping spec
```

Comment:
Only taking a quick look, and missing context, so wondering, just to double check: is the case of a "more-generic" spec (and therefore no warnings should be emitted) covered by other cases?

Reply:
No, it's not covered elsewhere. I didn't really want to add that because in a sense catching a more generic spec would be a good thing - it's just that we can't because of the false-positives it would create, so I just created a test for that situation.